imageproxy_test.go

```diff
@@ -349,7 +351,17 @@ func (t testTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 
 		raw = fmt.Sprintf("HTTP/1.1 200 OK\nContent-Length: %d\nContent-Type: image/png\n\n%s", len(img.Bytes()), img.Bytes())
 	default:
-		raw = "HTTP/1.1 404 Not Found\n\n"
+		redirectRegexp := regexp.MustCompile(`/redirects-(\d)`)
```

Comment:
I've got a local change that cleans up this testTransport type so that we're not shoving all these random things into it like this.  This is fine for now, but I'll likely refactor this later.